src/getBullsAndCows.js

```diff
@@ -0,0 +1,30 @@
+/* eslint-disable no-console */
```

Comment:
Disabling eslint is not a good practice, it should be avoided unless it is absolutely necessary. Try to adjust your code to meet the linting standards.

Reply:
thank you for review!